examples/plot_steps_nowcast.py

```diff
@@ -8,187 +8,215 @@
 
 """
```

Comment:
I also support the idea of a dedicated example script showing how to use the logging module.

Reply:
Using the logging module instead of print you don't see any output in the console if you don't add the proper config inside the example.
I can revert the improvements (ex: `main` function) but at minimum I need to add the `logging.basicConfig` at the top of the examples.

Reply:
We can define a default configuration in the `pysteps.__init__.py` that prints the INFO messages by default and add a set_loglevel function to pysteps [as in matplotlib](https://matplotlib.org/3.1.1/faq/troubleshooting_faq.html#getting-help). 

This is an example using [matplotlib's logging implementation](https://github.com/aperezhortal/logging_test) that logs the info messages by default.

Reply:
About that there is an open issue: https://github.com/matplotlib/matplotlib/pull/13275
Personally I dislike and I'm not sure it's a good way to do the work.

Reply:
I have not found any other library that uses `logging` like matplotlib.
It's also true that I don't find any library that uses so many output messages like pySTEPS to be able to compare the code and evaluate how they implement the output during a normal run.

Reply:
Although it is not the same as matplotlib, TensorFlow seems to implement a default handler and an interface to control the logging level.

https://github.com/tensorflow/tensorflow/blob/e5bf8de410005de06a7ff5393fafdf832ef1d4ad/tensorflow/python/platform/tf_logging.py#L93-L143

```
import logging
logger = tf.get_logger()
logger.setLevel(logging.ERROR)
```